File: API-Sebrae.R
install.packages(c("httr", "jsonlite"))

result <- httr::POST(
  url    = "https://apiplataforma.sebrae.com.br/graphql",
  body   = "{\"query\":\"{getEmpresasAleatorias(qtdElementos: 10)}\"}", 
  config = httr::config(ssl_verifypeer = FALSE)
)

parsed <- jsonlite::fromJSON(content(result, "text"), simplifyVector = TRUE)

File: API-Sebrae.py
import requests

url = "https://apiplataforma.sebrae.com.br/graphql"

payload = "{\"query\":\"{getEmpresasAleatorias(qtdElementos: 10)}\"}"
headers = {
    'Content-Type': "application/json",
    }

response = requests.request("POST", url, data=payload, headers=headers, verify=False)

print(response.text)
